core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/move.kt

```diff
@@ -51,39 +220,152 @@ public fun <T> DataFrame<T>.moveTo(newColumnIndex: Int, vararg columns: KPropert
+@ExcludeFromSources
+private interface SetMoveToStartOperationArg
+
+/**
+ * {@include [MoveToStart]}
+ * ### This MoveToStart Overload
+ */
+@ExcludeFromSources
+private interface CommonMoveToStartDocs
+
+@Deprecated(MOVE_TO_LEFT, ReplaceWith(MOVE_TO_LEFT_REPLACE), DeprecationLevel.ERROR)
+public fun <T> DataFrame<T>.moveToLeft(columns: ColumnsSelector<T, *>): DataFrame<T> = move(columns).toStart()
 
+/**
+ * @include [CommonMoveToStartDocs]
+ * @include [SelectingColumns.Dsl.WithExample] {@include [SetMoveToStartOperationArg]}
+ * @param [columns] The [Columns Selector][ColumnsSelector] used to select the columns of this [DataFrame] to move.
+ */
 @Refine
-@Interpretable("MoveToLeft1")
-public fun <T> DataFrame<T>.moveToLeft(columns: ColumnsSelector<T, *>): DataFrame<T> = move(columns).toLeft()
+@Interpretable("MoveToStart1")
+public fun <T> DataFrame<T>.moveToStart(columns: ColumnsSelector<T, *>): DataFrame<T> = move(columns).toStart()
+
+@Deprecated(MOVE_TO_LEFT, ReplaceWith(MOVE_TO_LEFT_REPLACE), DeprecationLevel.ERROR)
+public fun <T> DataFrame<T>.moveToLeft(vararg columns: String): DataFrame<T> = moveToStart { columns.toColumnSet() }
 
-public fun <T> DataFrame<T>.moveToLeft(vararg columns: String): DataFrame<T> = moveToLeft { columns.toColumnSet() }
+/**
+ * @include [CommonMoveToStartDocs]
+ * @include [SelectingColumns.ColumnNames.WithExample] {@include [SetMoveToStartOperationArg]}
+ * @param [columns] The [Columns Selector][ColumnsSelector] used to select the columns of this [DataFrame] to move.
+ */
+public fun <T> DataFrame<T>.moveToStart(vararg columns: String): DataFrame<T> = moveToStart { columns.toColumnSet() }
 
+@Deprecated(MOVE_TO_LEFT, ReplaceWith(MOVE_TO_LEFT_REPLACE), DeprecationLevel.ERROR)
 @AccessApiOverload
 public fun <T> DataFrame<T>.moveToLeft(vararg columns: AnyColumnReference): DataFrame<T> =
-    moveToLeft { columns.toColumnSet() }
+    moveToStart { columns.toColumnSet() }
 
+@AccessApiOverload
+public fun <T> DataFrame<T>.moveToStart(vararg columns: AnyColumnReference): DataFrame<T> =
+    moveToStart { columns.toColumnSet() }
+
+@Deprecated(MOVE_TO_LEFT, ReplaceWith(MOVE_TO_LEFT_REPLACE), DeprecationLevel.ERROR)
 @AccessApiOverload
 public fun <T> DataFrame<T>.moveToLeft(vararg columns: KProperty<*>): DataFrame<T> =
-    moveToLeft { columns.toColumnSet() }
+    moveToStart { columns.toColumnSet() }
+
+@AccessApiOverload
+public fun <T> DataFrame<T>.moveToStart(vararg columns: KProperty<*>): DataFrame<T> =
+    moveToStart { columns.toColumnSet() }
 
 // endregion
 
-// region moveToRight
+// region moveToEnd
 
+/**
+ * ## The MoveToEnd Operation
+ *
+ * Moves the specified [columns] to the [DataFrame] end.
+ * Returns a new [DataFrame] with updated columns order.
+ *
+ * @include [SelectingColumns.ColumnGroupsAndNestedColumnsMention]
+ *
+ * See [Selecting Columns][MoveToEndSelectingOptions].
+ *
+ * For more information: {@include [DocumentationUrls.Move]}
+ */
+internal interface MoveToEnd {
```

Comment:
I probably couldn't help myself and create a generic MoveToX doc hahaha, but that would probably be overkill, great job :)

Reply:
done